stage #1/module #6. IO, NIO/lesson #1. IO/content.md

```diff
@@ -0,0 +1,127 @@
+# Java I/O (java.io)
+
+## Introduction to IO
+
+Java I/O (Input and Output) is used to process the input and produce the output.
+Java uses the concept of a stream to make I/O operation fast. The java.io package contains all the classes 
```

Comment:
Could you wrap package with `? For example `java.io`.

Reply:
Done